packages/models/src/actor/interface.ts

```diff
@@ -0,0 +1,28 @@
+import { Behavior } from '../utils';
+
+export type ActorURI = string;
+
+// TODO:
+export interface ActorRef {
+  name: symbol | string;
+  protocol: string;
+  path: string;
+  uri: ActorURI;
+}
+
+export type MessagePayload<Payload = string | number | object> = {
+  symbol: symbol;
+  value?: Payload;
```

Comment:
The `symbol` in the `MessagePayload` works as a signal of the message and will be pattern matched by `@HandleCall(pattern)` in an actor class. It can be thought of as a `uri` and will be routed by the framework to a matched method.

A rough solution is in the example: https://github.com/ckb-js/kuai/pull/29/files#diff-ac37e008865a4a1a50830983363621a22689b22270fdf155d70eaeb0269beebdR38-R53